test/commands/match-neutrino-filter-address.cpp

```diff
@@ -49,11 +49,11 @@ BOOST_AUTO_TEST_CASE(match_neutrino_filter_address__invoke__no_match)
         "ff0424eb3cfd00000000230db414c859a07e8205876354"
         "a210a75042d0463404913d61a8e068e58a3ae2aa080026");
 
-    command.set_filter_argument({ filter });
+    command.set_compact_filter_argument({ filter });
     const std::string address("mfXcDNV1r9SPmNVEba34DG7CWrLEaRzKm3");
     command.set_address_argument({ address });
     BX_REQUIRE_INVALID(command.invoke(output, error));
-    BX_REQUIRE_OUTPUT("Address did not match filter.\n");
+    BX_REQUIRE_OUTPUT("Address does not match filter.\n");
```

Comment:
Constants should be used here. This already produced a break (which has been updated).